src/pages/main.py

```diff
@@ -281,35 +281,47 @@ def get_data(config):
 
 
 def main(session_state):
-    #  ==== GOOGLE ANALYTICS SETUP ====
-    GOOGLE_ANALYTICS_CODE = os.getenv("GOOGLE_ANALYTICS_CODE")
-    if GOOGLE_ANALYTICS_CODE:
-        import pathlib
-        from bs4 import BeautifulSoup
-
-        GA_JS = (
+    if os.getenv("IS_LOCAL").upper() != "TRUE":
```

Comment:
sugestão aqui: criar uma variável de ambiente `IS_DEV = TRUE` para usar ao invés do `IS_LOCAL`, diferenciando quando está no stagging x production.

Reply:
vejo com o thenets se da para deixar `TRUE` no heroku e `FALSE` no oficial

Reply:
Se for assim basta então que o if fique "if os.getenv("IS_DEV ").upper() == "FALSE":" para subir pois assim nao pega nem stagging nem local, pode ser?

Reply:
Aí voce teria que colocar no arquivo da ENV a variavel IS_DEV correto?